tests/translate/overrides/standard.yaml

```diff
@@ -1,8 +1,3 @@
-C_SW:
-  - backend: numpy
-    platform: docker
-    max_error: 2e-10
```

Comment:
removing this because it was only added as a demonstration of the format before we had overrides